src/styles/page.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  display: flex;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
The styles for `.stars__star` should be moved to a separate CSS file dedicated to the `stars` BEM block, as per the task requirements.